package.json

```diff
@@ -29,7 +29,7 @@
     "yamljs": "^0.3.0"
   },
   "peerDependencies": {
-    "fastify": ">=1.13.0"
+    "fastify": ">=2.0.0-rc.0"
```

Comment:
I think this might cause trouble down the line. We should remove the peerDependency completely, it's not really needed as it's checked by fastify-plugin